### PR TITLE
feat: Add timestampvm-rs as tempate

### PR DESCRIPTION
**Pull request by dimitrovmaksim**
Converts the timestampvm-rs code into a `cargo generate` template.

### CODE REVIEW

**Review by richardpringle — APPROVED** [review]

File: .github/workflows/test-and-release.yml
Comment (richardpringle) [inline .github/workflows/test-and-release.yml:1]: Can we remove this file from the template? Testing here will be a little different than normal

File: .rustfmt.nightly.toml
Comment (richardpringle) [inline .rustfmt.nightly.toml:1]: Can we remove this file? It was related to CI

File: README.md
Comment (richardpringle) [inline README.md:1]: Can we update the readme to accurately describe how to use `cargo generate` (including how to install `cargo generate`)? Then we can point to a permalink to `timestampvm-rs`.

File: cargo-generate.toml
Comment (richardpringle) [inline cargo-generate.toml:1]: 👍

File: deleteme
Comment (richardpringle) [inline deleteme:1]: nice haha, thank you!

File: scripts/tests.lint.sh
Comment (richardpringle) [inline scripts/tests.lint.sh:1]: Can delete this file

File: scripts/tests.unit.sh
Comment (richardpringle) [inline scripts/tests.unit.sh:1]: Can delete this as well, people should know how to use `cargo test`

File: scripts/tests.unused.sh
Comment (richardpringle) [inline scripts/tests.unused.sh:1]: Can delete this as well

**Review by richardpringle — CHANGES_REQUESTED** [review]
Just need to delete a couple of files then this is good to go

**Comment by dimitrovmaksim** [comment]
```
PROTOC_VERSION=$(protoc --version | cut -f2 -d' ')
if [[ "${PROTOC_VERSION}" == "" ]] || [[ "${PROTOC_VERSION}" < 3.15.0 ]]; then
  echo "protoc must be installed and the version must be greater than 3.15.0"
  exit 255
fi
```

This condition in the build script is not working for me
```
$ protoc --version
libprotoc 25.0
```

But still the ```[[ "${PROTOC_VERSION}" < 3.15.0 ]]``` conditions results to false

File: README.md
Comment (dimitrovmaksim) [reply README.md:1]: Updated the readme, but left the build and e2e test steps and added links to the timestampvm-rs repo

**Review by richardpringle — APPROVED** [review]
🥇

**Pull request merged**